align/cell_fabric/grid.py

```diff
@@ -119,8 +122,9 @@ def __init__( self, *, colors, pitch, width, offset=0, repeat=None):
         self.semantic()
 
 class EnclosureGrid(Grid):
-    def __init__( self, *, clg=None, pitch, offset=0, stoppoint, check=False):
-        assert not check or 2*stoppoint <= pitch
+    def __init__( self, *, clg=None, pitch, offset=0, stoppoint, check=True):
+        if check and 2*stoppoint > pitch:
+            logger.debug( f"2*Stoppoint is greater than pitch. Stoppoint={stoppoint}, Pitch={pitch}")
```

Comment:
We should change this comment to say something about the Enclosure grid stop point is more than half the pitch causing the physical coordinate to be non-monotonic with the grid ordering.  This is not inherently an issue, just confusing.

Reply:
Changed the message.